src/browser/main.coffee

```diff
@@ -118,7 +127,7 @@ parseCommandLine = ->
   options.boolean('portable').describe('portable', 'Set portable mode. Copies the ~/.atom folder to be a sibling of the installed Atom location if a .atom folder is not already there.')
   options.alias('t', 'test').boolean('t').describe('t', 'Run the specified specs and exit with error code on failures.')
   options.string('timeout').describe('timeout', 'When in test mode, waits until the specified time (in minutes) and kills the process (exit code: 130).')
-  options.alias('v', 'version').boolean('v').describe('v', 'Print the version.')
+  options.alias('v', 'version').boolean('v').describe('v', 'Print the version information.')
```

Comment:
I'm not totally sure, but I'm wondering if we'd want to keep the `--version` flag as-is, but then be able to modify it to display the full version info with a `--verbose` flag too. Maybe that's too convoluted, though.